.github/workflows/Pkg EdFi.Ods.Minimal.Template.PostgreSQL.yml

```diff
@@ -49,14 +60,23 @@ jobs:
          echo "StandardVersions=$output" >> $env:GITHUB_OUTPUT
          Write-host "StandardVersions is  $output"
        shell: pwsh
+     - name: Set ExtensionVersions
+       id: Set_ExtensionVersions
+       working-directory: ./Ed-Fi-Extensions/
+       run: |
+         $output = .\build.githubactions.ps1 ExtensionVersions -ProjectFile "$env:GITHUB_WORKSPACE/Ed-Fi-Extensions/Extensions/EdFi.Ods.Extensions.TPDM/EdFi.Ods.Extensions.TPDM.csproj"
+         echo "ExtensionVersions=$output" >> $env:GITHUB_OUTPUT
+         Write-host "ExtensionVersions is  $output"
```

Comment:
Would it make sense to be consistent here? `echo` and `Write-host` are the same command, so it seems strange to see them both here side by side.